src/workers/data-importer.ts

```diff
@@ -32,9 +32,9 @@ interface IndexProperty {
 }
 
 type AnyContiguousData = { id: string };
-type UnbundleableItem = (NormalizedDataItem | PartialJsonTransaction) &
+type TxWithParentIndex = (NormalizedDataItem | PartialJsonTransaction) &
```

Comment:
I think the original name for this is fine. We can only unbundle items with indexes, and they can be either data items or transactions so we shouldn't include `Tx` in the name.

A comment explaining that `Ans104Parser` requires items with indexes would be helpful though for clarity.

Reply:
reverted with comments as mentioned